docs/layers/accounts/tutorials/how-to-create-and-setup-aws-accounts.mdx

```diff
@@ -6,203 +6,294 @@ description: Guide for creating and setting up new AWS accounts
+import StepNumber from '@site/src/components/StepNumber';
+import Note from '@site/src/components/Note';
+import AtmosWorkflow from '@site/src/components/AtmosWorkflow';
 
 <Intro>
 This guide covers the process of creating and setting up new AWS accounts. It includes detailed instructions for updating account catalogs, configuring stack settings, deploying necessary components, and managing AWS account profiles.
 </Intro>
 
 ## Problem
 
-New accounts can be daunting to setup
+Setting up new AWS accounts can be complex and error-prone without proper guidance and tooling.
 
 ## Solution
 
-### PR 1 - Create account
+This guide provides a step-by-step approach to create and configure new AWS accounts using Cloud Posse's reference architecture components and conventions.
 
-#### Stack account catalog
+<Steps>
+  <Step>
+    ### <StepNumber/> Update Account Catalog
 
-See the `stack/catalog/account*.yaml`
+    First, add the new account to your account catalog configuration. This defines the account structure within your AWS Organization.
 
-If the new account is `pca`
+    Navigate to your account catalog file (typically `stacks/catalog/account.yaml`) and add the new account under the appropriate Organizational Unit (OU).
 
-Under the appropriate OU
+    **Example for a new account named `foobar`:**
 
-```
-          organizational_units:
-            - name: mgmt
-              accounts:
-```
+    ```yaml
+    organizational_units:
+      - name: core
+        accounts:
+          # ... existing accounts ...
+          - name: foobar
+            stage: foobar
+            tags:
+              eks: false
+    ```
 
-Add the following
+    <Note title="Important">
+      Choose the appropriate OU based on your organization's account strategy. Common OUs include `core` for core infrastructure and `plat` for platform accounts.
+    </Note>
+  </Step>
+
+  <Step>
+    ### <StepNumber/> Create Stack Configuration Files
+
+    Create the necessary stack configuration files for your new account using the `stacks/orgs/<namespace>/<tenant>/<stage>` pattern. See the following example where the namespace is `acme`, the tenant is `core`, and the stage is `foobar`.
+
+    **Example Directory Structure:**
+
+    For a new account `foobar` in the `core` tenant under namespace `acme`:
+
+    ```
+    stacks/orgs/acme/core/foobar/
+    ├── _defaults.yaml
+    ├── global-region/
+    │   ├── baseline.yaml          # Account settings
+    │   └── identity.yaml          # AWS Team Roles
+    └── us-east-1/
+        ├── baseline.yaml          # Regional baseline
+        └── app.yaml               # Regional application platform
+    ```
+
+    <Note title="Recommendation">
+      Start with a basic global-region stack for account settings and create placeholder regional stacks for future expansion. You can add more regional stacks and components as your account requirements grow.
+    </Note>
+
+    <Steps>
+      <Step>
+        ### <StepNumber/> Stage Mixin
+
```

Comment:
```suggestion
  By convention, we treat every account as an operating stage. Stages are configured as mixins, so that each stack operating in that stage can import that mixin to have a consistent stage name.
```